models/sushiswap/bnb/sushiswap_bnb_trades.sql

```diff
@@ -0,0 +1,159 @@
+           s1.call_tx_hash                                                    AS call_tx_hash,
+           s1.contract_address                                                AS contract_address,
+           s1.deadline                                                        AS deadline,
+           s1.output_amounts                                                  AS output_amounts,
+           s1.path                                                            AS path,
+           s1.to                                                              AS to,
+           --swapTokensForExactETH
+           case when s2.amountOut = 0 then s2.amountOut else s2.amountOut end AS token_sold_amount_raw,
+           s2.amountInMax                                                     AS token_sold_amount_max,
+           s2.call_block_number                                               AS call_block_number,
+           s2.call_block_time                                                 AS call_block_time,
+           s2.call_success                                                    AS call_success,
+           s2.call_trace_address                                              AS call_trace_address,
+           s2.call_tx_hash                                                    AS call_tx_hash,
+           s2.contract_address                                                AS contract_address,
+           s2.deadline                                                        AS deadline,
+           s2.output_amounts                                                  AS output_amounts,
+           s2.path                                                            AS path,
+           s2.to                                                              AS to,
+           --quote
+            q.amountA AS amountA,
+            q.call_block_number AS call_block_number,
+            q.call_block_time AS call_block_time,
+            q.call_success AS call_success,
+            q.call_trace_address AS call_trace_address,
+            q.call_tx_hash AS call_tx_hash,
+            q.contract_address AS contract_address,
+            q.output_amountB AS output_amountB,
+            q.reserveA AS reserveA,
+            q.reserveB AS reserveB,
+
+    FROM {{ source('sushi_bnb', 'UniswapV2Pair_evt_Swap') }} t
+    INNER JOIN {{ source('sushi_bnb', 'swapExactETHForTokens') }} s1
+        ON s1.contract_address = t.contract_address 
+    INNER JOIN {{ source('sushi_bnb', 'swapTokensForExactETH') }} s2
+        ON s2.contract_address = t.contract_address 
+    INNER JOIN {{ source('sushi_bnb', 'swapExactETHForTokensSupportingFeeOnTransferTokens') }} s3
+        ON s3.contract_address = t.contract_address
+        AND s3.call_block_number = s1.call_block_number
+        AND s3.call_tx_hash = s1.call_tx_hash
+        AND s3.amountETHMin = s1.amountOut
+    INNER JOIN {{ source('sushi_bnb', 'swapETHForExactTokens') }} s4
+        ON s4.contract_address = t.contract_address 
```

Comment:
These trade evt tables for `Router02` are actually need to be `UNION ALL`, not `INNER JOIN`.

Reply:
done